drivers/scheduler/k8s/k8s.go

```diff
@@ -630,6 +665,16 @@ func (k *k8s) WaitForRunning(ctx *scheduler.Context) error {
 			}
 
 			logrus.Infof("[%v] Validated StorkRule: %v", ctx.App.Key, svc.Name)
+		} else if obj, ok := spec.(*v1.Pod); ok {
+			isPodReady := k8sOps.IsPodReady(*obj)
```

Comment:
nit: change to `if ! k8sOps.IsPodReady(*obj) {`

Reply:
Fixed.